python/tidy/servo_tidy_tests/manifest-include.ini

```diff
@@ -0,0 +1,109 @@
+skip: true
+[_mozilla]
+  skip: false
+[2dcontext]
+  skip: false
+[cors]
+  skip: false
+[css]
+  skip: true
+  [compositing]
+    skip: false
+  [CSS2]
+    skip: false
+  [css-animations]
+    skip: false
+  [css-backgrounds]
+    skip: false
+  [css-color]
+    skip: false
+  [css-conditional]
+    skip: false
+  [css-flexbox]
+    skip: false
+  [css-images]
+    skip: false
+  [css-style-attr]
+    skip: false
+  [css-text]
+    skip: false
+  [css-text-decor]
+    skip: false
+  [css-transforms]
+    skip: false
+  [css-transitions]
+    skip: false
+  [css-ui]
+    skip: false
+  [css-values]
+    skip: false
+  [css-variables]
+    skip: false
+  [cssom]
+    skip: false
+  [cssom-view]
+    skip: false
+  [filter-effects]
+    skip: false
+  [geometry]
+    skip: false
+  [mediaqueries]
+    skip: false
+  [selectors]
+    skip: false
+[custom-elements]
+  skip: false
+[dom]
+  skip: false
+[domparsing]
+  skip: false
+[encoding]
+  skip: false
+[eventsource]
+  skip: false
+[fetch]
+  skip: false
+[FileAPI]
+  skip: false
+[fullscreen]
+  skip: false
+[hr-time]
+  skip: false
+[html]
+  skip: false
+[js]
+  skip: false
+[navigation-timing]
+  skip: false
+[old-tests]
+  skip: true
+  [submission]
+    skip: true
+[performance-timeline]
+  skip: false
+[quirks]
+  skip: false
+[referrer-policy]
+  skip: false
+[subresource-integrity]
+  skip: false
+[touch-events]
+  skip: false
+[uievents]
+  skip: false
+[url]
+  skip: false
+[WebCryptoAPI]
+  skip: false
+[webgl]
+  skip: false
+[WebIDL]
+  skip: false
+[websockets]
+  skip: false
+[webstorage]
+  skip: false
+[workers]
+  skip: false
+[xhr]
+  skip: false
```

Comment:
Rather than duplicating the manifest that currently works, we should have a minimized manifest like I described in my comment - one path is that is valid (like `html`, which is unlikely to ever change), and one that is invalid (like `never_going_to_exist`).